docs/resources/cached_image.md

```diff
@@ -27,6 +27,8 @@ resource "lxd_instance" "test1" {
 
 * `source_image` - *Required* - Fingerprint or alias of image to pull.
 
+* `type` - *Optional* - Type of image to cache. Must be one of `container` or `virtual-machine`
```

Comment:
I would just emphasize here that it defaults to `container`.

Reply:
Thanks. I noticed this yesterday while reviewing again on my phone and is actually what stopped me from clicking the merge button. :)